src/components/CatCatalogue.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from "react";
+
+import CatService from "../services/CatServices";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrashAlt, faTrash } from "@fortawesome/free-solid-svg-icons";
+
+function CatCatalogue() {
+  const pageSize = 10;
+  const [data, setData] = useState([]);
```

Comment:
As part of the exercise, we should use TypeScrip, so this file will become `CatCatalogue.tsx` (like this genius component name). And this will become something like:
```suggestion
  const [cats, setCats] = useState<CatSummary>([]);
```